README.md

```diff
@@ -77,3 +77,27 @@ The current NFT contract is deployed on the Mumbai testnet here: [0x35b5147E7499
 
 The current Token contract is deployed on the Mumbai testnet here: [0xc4C99f33F686A74a2Fe95B26Ce317708f605A9eA](https://mumbai.polygonscan.com/address/0xc4C99f33F686A74a2Fe95B26Ce317708f605A9eA)
 
+## Setup CI
```

Comment:
I would just call this "Setup Slither-Analyzer" instead of "Setup CI". The CI will still work, this is an optional step if you want to check your code yourself before creating a PR.